content/md/posts/2023-07-08-unit-testing.md

```diff
@@ -0,0 +1,66 @@
+{:title "Unit testing"
+ :layout :post
+ :tags  ["cljs" "testing"]
+ :toc true}
+
+
+## Overview
+
+This article covers what unit testing is, why it is useful, and an example of unit testing in a project.
+
+## What is unit testing?
+
+ Unit testing is about testing a function, using the inputs and verifying the outputs. Unlike acceptance testing which tests the result of a collection of functions interacting together, unit testing just grabs one function and sees whether it reacts correctly given certain inputs.
+ 
+
+## Why is it useful?
```

Comment:
Not sure what's the target, perhaps a better name would be `Why testing is useful?` ...

Reply:
I'll make the title more clear.